sound/soc/intel/boards/Makefile

```diff
@@ -41,9 +41,9 @@ snd-soc-sof-sdw-objs += sof_sdw.o				\
 			sof_sdw_rt5682.o sof_sdw_rt700.o	\
 			sof_sdw_rt711.o sof_sdw_rt_sdca_jack_common.o	\
 			sof_sdw_rt712_sdca.o sof_sdw_rt715.o	\
-			sof_sdw_rt715_sdca.o sof_sdw_dmic.o	\
-			sof_sdw_cs42l42.o sof_sdw_cs42l43.o	\
-			sof_sdw_cs_amp.o			\
+			sof_sdw_rt715_sdca.o sof_sdw_rt722_sdca.o	\
+			sof_sdw_dmic.o sof_sdw_cs42l42.o	\
+			sof_sdw_cs42l43.o sof_sdw_cs_amp.o	\
```

Comment:
best to keep some sort of order, e.g.
sof_sdw_rt715_sdca.o sof_sdw_rt722_sdca.o
sof_sdw_cs42l42.o sof_sdw_cs42l43.o	\
sof_sdw_cs_amp.o
sof_sdw_dmic.o
sof_sdw_hdmi.o